### ISSUE TITLE
[generic] The Guardian.com downloads javascript file (not video)

### ISSUE THREAD

**Issue by RingoTheDog** [rg3/youtube-dl#10780]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2016.09.27*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2016.09.27**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add `-v` flag to **your command line** you run youtube-dl with, copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
```
>youtube-dl.py -v "https://www.theguardian.com/football/2016/sep/26/leicester-defence-claudio-ranieri-porto-champions-league"
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', 'https://www.theguardian.com/football/2016/sep/26/leicester-defence-claudio-ranieri-porto-champions-league']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2016.09.27
[debug] Python version 3.5.1 - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-71727-g46778ab, rtmpdump 2.4
[debug] Proxy map: {}
[generic] leicester-defence-claudio-ranieri-porto-champions-league: Requesting header
WARNING: Falling back on generic information extractor.
[generic] leicester-defence-claudio-ranieri-porto-champions-league: Downloading webpage
[generic] leicester-defence-claudio-ranieri-porto-champions-league: Extracting information
[debug] Invoking downloader on 'https://assets.guim.co.uk/javascripts/bootstraps/enhanced/54d52f7dd1c2107bf2a8d9b3a2d0e8f3/profile.js'
[download] Destination: Leicester's Claudio Ranieri worried by defensive frailties but ready for Porto-profile.js
[download] 100% of 23.04KiB in 00:00
...
<end of log>
```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://www.theguardian.com/football/2016/sep/26/leicester-defence-claudio-ranieri-porto-champions-league

- Same video, diff URL:  https://www.theguardian.com/football/video/2016/sep/27/claudio-ranieri-leicester-city-porto-champions-league-video

---

### Description of your *issue*, suggested solution and other information

results in 24k .js file

thanks
Ringo

**Comment by remitamine** [comment]
work's with the latest version.

**Comment by yan12125** [comment]
I accidentally pushed the fix for this bug in bd2644120526429783c55e885f7042633826d7da. However, that commit brings more new bugs and I'm still handling them. Anyway thanks for cleaning up leftovers.
